### PR TITLE
Lesson 7 solutions

### PR DESCRIPTION
**Pull request by synulewar**
@kuszcjan

### CODE REVIEW

File: students/synowiec_krzysztof/lesson_07_string_datetime/to_swap_the_two_words.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+def main():
+    first, second = input().split()
+    print("{0} {1}".format(first, second))
```
Comment (kuszcjan) [inline students/synowiec_krzysztof/lesson_07_string_datetime/to_swap_the_two_words.py]: This does not swap the words, they are printed exactly the same as input.
@synulewar please fix.

**Comment by kuszcjan** [comment]
just took a quick look. no major problems found so far.

File: students/synowiec_krzysztof/lesson_07_string_datetime/pythonFile.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import sys
```
Comment (kuszcjan) [inline students/synowiec_krzysztof/lesson_07_string_datetime/pythonFile.py]: not sure which task this should solve. maybe was committed by mistake?

Comment (synulewar) [reply students/synowiec_krzysztof/lesson_07_string_datetime/pythonFile.py]: Yes, that was mistake. It is my script to create empty file with main function. I copy name from snakify or scope and it conver it name to small leters with _. :)

**Comment by synulewar** [comment]
@kuszcjan 
I pushed commit with fix and I can see this change in this pull request but I' m not sure github will refresh and notify.

File: students/synowiec_krzysztof/lesson_07_string_datetime/pythonFile.py (hunk 2 above)
Comment (kuszcjan) [reply students/synowiec_krzysztof/lesson_07_string_datetime/pythonFile.py]: hah, understood :)

**Comment by kuszcjan** [comment]
With push force, I don't get a notifications. 
I would prefer additional commit with fixes rather than overwriting single commit. ( and notifications should work too)

**Pull request merged**